src/test/java/net/snowflake/ingest/streaming/internal/ChannelCacheTest.java

```diff
@@ -4,27 +4,28 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import net.snowflake.ingest.streaming.OpenChannelRequest;
+import org.apache.arrow.vector.VectorSchemaRoot;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
 public class ChannelCacheTest {
```

Comment:
Thanks for the changes!
Thinking about the other PR where we also introduce the Parquet file format: Can we refactor (some) test classes so that we reuse the tests for Parquet format as well and not copy/paste the same cases?

Reply:
I think we should, I did not get to it yet as the disclaimer in the PR description says  :)

Reply:
the test refactoring is now in #209.